pkg/apis/apis/v1alpha1/types_apiexport.go

```diff
@@ -274,7 +274,9 @@ type APIExportStatus struct {
 	// +optional
 	Conditions conditionsv1alpha1.Conditions `json:"conditions,omitempty"`
 
-	// virtualWorkspaces contains all APIExport virtual workspace URLs.
+	// virtualWorkspaces (deprecated)  contains all APIExport virtual workspace URLs.
```

Comment:
Probably too early to do this :)

Reply:
I tend to think deprecate as early as possible, remove as late as possible. I will use the future tense to make things less scary.